tests/browser/donations_payments/create_donation_and_payment.robot

```diff
@@ -17,12 +17,11 @@ Create Donation from a Contact
     &{contact} =  API Create Contact    Email=skristem@robot.com
     &{opportunity} =  API Create Opportunity    &{Contact}[AccountId]    Donation    Name=Sravani $100 donation
     Go To Record Home  &{opportunity}[Id]
+    Select Tab    Related
     ${opp_name}    Get Main Header 
     Set Global Variable      ${opp_name}
     Load Related List    Payments
-    Select Related Dropdown    Payments
-    Wait For Locator    link-title    Schedule Payments
-    Click Link    link=Schedule Payments
+    Click Special Related List Button  Payments    Schedule Payments
     Wait For Locator    frame    Create one or more Payments for this Opportunity
     Select Frame with Title    Create one or more Payments for this Opportunity
```

Comment:
Looks like this one failed here: https://mrbelvedereci.herokuapp.com/tests/result/31661060